sweep/sweeper.go

```diff
@@ -282,6 +341,27 @@ func (s *UtxoSweeper) SweepInput(input input.Input) (chan Result, error) {
 	return sweeperInput.resultChan, nil
 }
 
+// validateFeePreference validates a client's fee preference in accordance to
+// the UtxoSweeper's config.
+func (s *UtxoSweeper) validateFeePreference(
```

Comment:
Name of the function only partially covers what it does. It is also used to pick a fee rate.

Reply:
Fixed.